.changeset/few-beds-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
+---
+
+Fixes chunk issues when bundling node_modules dependencies
```

Comment:
Fixes specific _but not all_ cases where the project can't in Cloudflare, if dependencies from `node_modules` are only used in prerendered pages. This change shouldn't have any side-effects.